Sources/ECMASwift/API/Headers/Headers.swift

```diff
@@ -41,8 +41,12 @@ final class Headers: NSObject, HeadersExports {
 /// Helper to register the ``Headers`` API with a context.
 struct HeadersAPI {
     func registerAPIInto(context: JSContext) {
-        let headersClass: @convention(block) () -> Headers = {
-            Headers()
+        let headersClass: @convention(block) (JSValue?) -> Headers = { headers in
+            if let headers {
```

Comment:
I think you'll also need to check if the underlying `JSValue` is also `null` or `undefined` here, there's a helper for this. Something like:

```swift
if let headers, headers.hasValue {
```

Reply:
sorry for not seeing this. did you make the change?

Reply:
Yep, all good, thanks again for the contribution!